cfn-resources/database-user/resource-role.yaml

```diff
@@ -31,6 +31,20 @@ Resources:
                 - "secretsmanager:UpdateSecretVersionStage"
                 - "ec2:CreateVpcEndpoint"
                 - "ec2:DeleteVpcEndpoints"
+                - "cloudformation:CreateResource"
+                - "cloudformation:DeleteResource"
+                - "cloudformation:GetResource"
```

Comment:
Why are these needed?

Reply:
These permissions are generic and the same will applied when we publsih, doesn't harm anything for private extension